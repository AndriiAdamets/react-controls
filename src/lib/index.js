import Button from './components/Button';
import Card from './components/Card';
import Tooltip from './components/Tooltip';
import Checkbox from './components/Checkbox';
import Input from './components/Input';
export { Button, Card, Tooltip, Checkbox, Input };