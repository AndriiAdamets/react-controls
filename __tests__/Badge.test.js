import React from 'react';
import renderer from 'react-test-renderer';
import { Badge } from '../src/lib';

test('Badge', () => {
  const component = renderer.create(
    <Badge>Badge</Badge>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Badge like pill', () => {
  const component = renderer.create(
    <Badge likePill={true}>Badge like pill</Badge>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});