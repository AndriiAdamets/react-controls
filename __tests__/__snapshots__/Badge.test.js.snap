// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge 1`] = `
<div
  className="wrc-badge wrc-badge--secondary"
>
  Badge
</div>
`;

exports[`Badge like pill 1`] = `
<div
  className="wrc-badge wrc-badge--secondary wrc-badge--pill"
>
  Badge like pill
</div>
`;
